README.md

```diff
@@ -787,6 +787,35 @@ Set this property to `false` to disable this plugin.
 _No options are available for this plugin._
 </details>
 
+### slideshow
+
+<details>
+<summary>Add support for Slideshow in Markdown, as block syntax.</summary>
+
+The basic syntax is `[slideshow <url1> <url2>]`. E.g., `[slideshow https://assets.digitalocean.com/banners/python.png https://assets.digitalocean.com/banners/javascript.png]`.
```

Comment:
```suggestion
The basic syntax is `[slideshow <url1> <url2> <...urls>]`. E.g., `[slideshow https://assets.digitalocean.com/banners/python.png https://assets.digitalocean.com/banners/javascript.png https://assets.digitalocean.com/banners/nodejs.png]`.
```
For the sake of making it more apparent that its a slideshow, I think it'd be good to have three images in the demo?

Reply:
Updated with the extra image.